src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -150,9 +150,11 @@ private void testEditSessionAction() throws Exception {
         assertEquals(Const.StatusCodes.BACKDOOR_STATUS_SUCCESS, status);
 
         feedbackEditPage = getFeedbackEditPage();
-        feedbackEditPage.isElementSelected(Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON
-                                           + "_atvisible");
-        //feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditPublished.html");
+
+        // Ensuring that the settings did not default back to original values after manual publishing
+        assertFalse(feedbackEditPage.isElementSelected(Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON
+                                                       + "_atvisible"));
+
```

Comment:
Just below this (I don't think it's being shown in the diff), there is this "restore defaults" that clicks back on the so called "default button" I believe that is the immediate visible button that is being asserted to not be selected right here. Originally, the session was immediately visible but was edited from above to be something else.

Right above here, there is this verifyHtmlMainContent after the session has been edited. I believe the original intention of the test was to ensure that the "visibility option" stays as the edited option.

However, previously this test was doing feedbackEditPage.isElementSelected() which returns a boolean but does no assertion so I added the assertion.